### PR TITLE
Made windows related dependencies optional

### PR DESCRIPTION
**Pull request by newpavlov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # fs2
 
 Extended utilities for working with files and filesystems in Rust. `fs2`
-requires Rust stable 1.1 or greater.
+requires Rust stable 1.8 or greater.
```
Comment (danburkert) [inline README.md]: Could you also bump the travis version?

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fs2"
-version = "0.2.5"
+version = "0.3.0"
```
Comment (danburkert) [inline Cargo.toml]: Don't make this change; I'll bump it when I make a new release.

File: Cargo.toml [hunk 3]
```diff
@@ -9,7 +9,9 @@ description = "Cross-platform file locks and file duplication."
 keywords = ["file", "file-system", "lock", "duplicate", "flock"]
 
 [dependencies]
-libc = "0.2.2"
+libc = "0.2"
```
Comment (danburkert) [inline Cargo.toml]: I can't recall exactly, but I think there was a reason libc "0.2.1" was incompatible.

**Comment by newpavlov** [comment]
Fixed all your comments. In addition to bumping 1.1 to 1.8 in travis config I've also added stable release.

Regarding making `libc` optional instead of using 'cfg(not(windows))' I've used 'cfg(unix)' as it's how you do imports in `lib.rs`.

**Comment by danburkert** [comment]
Perfect, thanks!

**Comment by danburkert** [comment]
(I'll do a release shortly)

**Pull request merged**